spec/binary/config_spec.rb

```diff
@@ -290,11 +290,11 @@ def assert_same_flat_opts(args1, args2)
       let(:lib_path) { File.expand_path '../../../lib', __FILE__ }
 
       it 'defaults to sib\'s lib path' do
-        expect(parse([]).lib_options.load_path_dirs).to eq [lib_path]
+        expect(parse([]).lib_options.load_path_dirs).to eq [File.realpath(lib_path)]
       end
 
       specify '-I and --load-path add their arguments to it' do
-        expect(parse(%w[-I f1 --load-path f2]).lib_options.load_path_dirs).to eq [lib_path, 'f1', 'f2']
+        expect(parse(%w[-I f1 --load-path f2]).lib_options.load_path_dirs).to eq [File.realpath(lib_path), 'f1', 'f2']
```

Comment:
This change actually has nothing to do with Windows compatibility. It's just there because my development directory is symlinked, and that fact was causing the test to fail when they would compare file paths.